harrier/common.py

```diff
@@ -59,6 +59,21 @@ def slugify(title):
     return name.strip('_-')
 
 
+ALLOWED_TITLE_VALS = re.compile(r'[^a-z0-9_\-]')
+HEADING_RE = re.compile(r'&(?:#\d{2,}|[a-z0-9]{2,});')
+
+
+def heading_slugify(title):
```

Comment:
I still don't understand why this can't just modify `slugify` above?

Who really cases about having  dots in slugs? also the removal of `&#39;` would be good in both.